lib/tasks/import/mobility_devices.rake

```diff
@@ -6,7 +6,7 @@ desc 'Import mobility devices from Access CSV at data/mobilitydevice.csv'
 namespace :import do
   task mobility_devices: :environment do
     CSV.foreach('data/mobilitydevice.csv', headers: true).with_index 2 do |row, line|
-      device = MobilityDevice.new device: row.fetch('name')
+      device = MobilityDevice.new name: row.fetch('name')
```

Comment:
Why didn't you have to change this from `row.fetch('device')` to what it is now?

Reply:
Ohhhh. Never mind.